.travis.yml

```diff
@@ -88,7 +92,7 @@ matrix:
           - clang
           - libstdc++-4.8-dev
           - libgmp-dev
-    - env: BUILD_TYPE="Debug" WITH_SYMENGINE_ASSERT="yes" WITH_SYMENGINE_RCP="no" WITH_PYTHON="yes" PYTHON_VERSION="2.7"
+    - env: BUILD_TYPE="Debug" WITH_BFD="yes" WITH_SYMENGINE_ASSERT="yes" WITH_PYTHON="yes" PYTHON_VERSION="2.7"
```

Comment:
I think you'll have to remove WITH_BFD on a Mac, as Travis fails due to this. Looks like the bfd library is there but not `link.h`. There might be some other way to implement it on a Mac, and we can look into it later, as there are many people who use OS X as their main development platform. For now I would just disable it.